### PR TITLE
Last changes v090

### PR DESCRIPTION
**Pull request by oualib**
- adding more package in the installation to avoid import issues
 - cleaning the code
 - new function set_connection to set a customised connection
 - new name vdf_from_relation -> vDataFrameSQL
 - Possibility to create a vDataFrame from a sql query using the new parameter sql.
 - hchart: possibility to use variables
 - new file logo to generate the verticapy logo
 - asfreq is renamed interpolate
 - New option for the set_option function: 'overwrite_model' to automatically overwrite models when creating a new one.
 - help_start function is updated with new links.

TODO for the next PR to finalise 0.9.0:
 - updating README.md
 - updating the entire documentation

### CODE REVIEW

**Review by ansleis — APPROVED** [review]

File: verticapy/utilities.py [hunk 1]
```diff
@@ -1657,15 +1792,17 @@ def set_option(option: str, value: Union[bool, int, str] = None):
...
                 verticapy.options["temp_schema"] = str(value)
             else:
-                raise ParameterError(
-                    "The schema '{}' could not be found.".format(value)
-                )
+                raise ParameterError(f"The schema '{value}' could not be found.")
     elif option == "time_on":
         check_types([("value", value, [bool])])
-        if isinstance(value, bool):
+        if value in (True, False, None):
```
Comment (afard) [inline verticapy/utilities.py:1798]: Just wondering, doesn't `check_types([("value", value, [bool])])` implies that `value` is in `(True, False, None)`?

Comment (oualib) [reply verticapy/utilities.py:1798]: It only raises a warning and not an error.

File: verticapy/learn/vmodel.py [hunk 2]
```diff
@@ -3255,15 +3257,15 @@ def predict_proba(
                 ("X", X, [list]),
                 ("vdf", vdf, [str, vDataFrame]),
                 ("inplace", inplace, [bool]),
-                ("pos_label", pos_label, [bool]),
+                ("pos_label", pos_label, [int, str, float]),
```
[the remark this answers is not in the document]
Comment (oualib) [reply verticapy/learn/vmodel.py:3260]: Yes 1.0 or 0.0 could work.

**Review by afard — APPROVED** [review]
Looks good to me. Please make sure the tests pass before merging.

**Pull request merged**